presentation/chapters/es-ES/unsafe.chapter

```diff
@@ -0,0 +1,65 @@
+## Unsafe Rust
+
+El sistema de tipo de Rust provee muchas garantías, pero en ocasiones esto conlleva que soluciones específicas sean difíciles o imposibles.
+
+Por esta razón, Rust tiene el concepto de "código inseguro".
+
+---
+
+El código inseguro tiene permitido:
+-   acceder memoria libremente
+-   utilizar apuntadores crudos (raw pointers)
+-   llamar a funciones externas
+-   declarar valores `Send` y `Sync`
+-   escribir a variables globales no sincronizadas
+
+---
+
+No es seguro:
+
+-   conversión a apuntadores crudos
```

Comment:
`crudo` is too related to food, I would take this option https://es.wikipedia.org/wiki/Puntero_(inform%C3%A1tica)#Haciendo_m.C3.A1s_seguros_a_los_punteros, (although the page indicates translation is not accurate :)). They use `punteros sin procesar`:

>>> En este contexto, los punteros que se dirigen directamente a la memoria (como se utilizan en este artículo) se denominan punteros sin procesar, en contraste con los punteros inteligentes u otras variantes